app/components/planning_applications/assessment_report_component.html.erb

```diff
@@ -66,14 +66,29 @@
 <% end %>
 <section id="policy-classes-section" class="govuk-!-padding-bottom-5">
   <h3 class="govuk-heading-s"><%= t(".legislation") %></h3>
-  <% if policy_classes.any? %>
-    <% policy_classes.each do |policy_class| %>
-      <%= render(
-        PolicyClasses::SummaryComponent.new(policy_class: policy_class)
-      ) %>
+  <% if @planning_application.application_type.name == "planning_permission" %>
```

Comment:
I don't know how policy classes versus policy areas work precisely, but I wonder if we could do without checking the application type here.

Something like (pseudocode)

```
if policy_area.present?
  considerations.each
    …
  end
elif policy_classes.present?
  policy_classes.each
    …
  end
end
```

Reply:
Mm it's a bit of a nightmare. Essentially they're the same thing but we can't load all of the law into bops so we've got a half way house. Not sure the naming/structure is the right way to go tbh 😬